pip_chill/cli.py

```diff
@@ -41,11 +41,30 @@ def main():
         dest="verbose",
         help="list commented out dependencies too.",
     )
+    parser.add_argument(
+        "-c",
+        "--conda",
+        action="store_true",
+        dest="conda",
+        help="format output for conda.",
+    )
     args = parser.parse_args()
 
     distributions, dependencies = pip_chill.chill(
         show_all=args.show_all, no_chill=args.no_chill
     )
+
+    if args.conda:
+        import os 
+        import sys 
+        print('name: {}'.format(os.path.split(sys.prefix)[1]))
+        if "CONDA_DEFAULT_ENV" in os.environ:
+            print(os.popen("conda config --show channels").read())
```

Comment:
Would be a good idea to verify whether conda is installed and/or wrap this in a try/except block in case it fails to run.